swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, condition=0.0, age=0):
+        super().__init__(condition, age, category="Clothing")
```

Comment:
Nice job inserting the `category="Clothing"` attribute directly in the super's initializer!